### DISCUSSION TITLE
App Language Screen Show Blank

### DISCUSSION THREAD

**Discussion by durgesh017** [oppia/oppia-android#5368] [Q&A (Navigating the codebase)] [answered]
@amire80 
@Bontrey 
@bladebunny
@sierisimo 
@yurilev

"I've been using the Oppia application and encountered an issue. After facing this issue, I feel it's very important to address, as it negatively affects the user's first impression. Do you have any solution for fixing this issue? I have a thought on how to fix this problem. May I propose my solution in this chat?"

Feel free to share your proposal, and I'll be happy to help further!

<img src="https://github.com/oppia/oppia-android/assets/101611806/e54bc660-048f-4736-90f7-1396e6c24e7e" width="200" height="400">
<img src="https://github.com/oppia/oppia-android/assets/101611806/7b17b010-12e0-4959-9758-0e6b4ee066c6" width="200" height="400">

**Comment by MohitGupta121** [comment]
@555Durgesh this is not a issue, you need to enable this screen in Platforms_Parameters

**Reply by durgesh017** [reply]
@MohitGupta121 can you share file name

**Reply by MohitGupta121** [reply]
https://github.com/oppia/oppia-android/blob/develop/utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

**Comment by durgesh017** [comment]
@MohitGupta121 
I Share Screenshot there not show platform parameter package
![Screenshot (235)](https://github.com/oppia/oppia-android/assets/101611806/5ca80fb0-d5d6-49a0-81b1-e73856c70841)

**Reply by MohitGupta121** [reply]
That is now removed in latest PR.

**Comment by Vishwajith-Shettigar** [comment, marked as answer]
Hi @555Durgesh, it seems that the app language screen is blank, possibly because the app was built using Gradle. If you check the production version of the app, it works perfectly fine. However, if you build the app using Bazel, you'll see the app language option screen.

**Reply by Vishwajith-Shettigar** [reply]
And this is not an issue.

**Reply by Vishwajith-Shettigar** [reply]
To get started with Bazel, please see [Oppia-Bazel-Setup-Instructions](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions) and follow the instructions appropriate to your OS.

**Comment by Rd4dev** [comment]
Hi @555Durgesh, as @Vishwajith-Shettigar said, the issue is indeed related to the Gradle build. 

To elaborate further, please refer to Issue #5106, where the [comment](https://github.com/oppia/oppia-android/issues/5106#issuecomment-1651323379) mentions that 

> **"By design, this language functionality is not available on Gradle, but you can see it if you build the app with Bazel."**

 I've been working with the Bazel build and can confirm its availability

**Screenshot of the Bazel Build Version:**
![Bazel Build](https://github.com/oppia/oppia-android/assets/122200035/c5e09cf7-de4b-4f05-b200-90735b6db72c)
